### PR TITLE
Make omise.js compatible with Browserify

### PR DESCRIPTION
**Pull request by wingyplus**
export Omise object when found module.exports object

### CODE REVIEW

File: src/omise.js [hunk 1]
```diff
@@ -61,6 +61,10 @@
...
   // exports
 
+  if (typeof module.exports == "object") {
+    module.exports = Omise;
+  }
+
   window.Omise = Omise;
 
-})(window);
+})(window, typeof module == "object" ? module : {});
```
Comment (sirn) [inline src/omise.js]: Can you use `===` here and in `typeof module.exports` above?

**Comment by sirn** [comment]
Thank you for your contribution! Can you do a quick fix above and we'll merge it ASAP. Thanks!

**Comment by wingyplus** [comment]
@sirn I fixed it.

**Comment by sirn** [comment]
Thanks!

**Pull request merged**